backstage/app-config.yaml

```diff
@@ -113,3 +98,38 @@ catalog:
             minutes: 60
           timeout:
             minutes: 30
+    ldapOrg:
+      default:
+        target: ldap://ralfed.cclrc.ac.uk
+        bind:
+          dn: uid=ldap-reader-user,ou=people,ou=example,dc=example,dc=net
+          secret: dW5hdXRoZW50aWNhdGVkIExEQVAK
```

Comment:
Where does this come from?

Reply:
It is a fake secret because the field has to be provided, but the server is unauthenticated.